core/src/main/java/org/elasticsearch/cluster/action/index/NodeMappingRefreshAction.java

```diff
@@ -85,7 +83,6 @@ public NodeMappingRefreshRequest() {
         public NodeMappingRefreshRequest(String index, String indexUUID, String[] types, String nodeId) {
```

Comment:
good catch, this caught another place where I should have moved things to be at the index level